File: Course_01/Java Language/Estrutura sequencial/pagamento.java
package pagamento;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		
		Scanner sc = new Scanner(System.in);
		
		String nome;
		double valor, total;
		int horas;
		
		System.out.print("Nome: ");
		nome = sc.nextLine();
		System.out.print("Valor por hora: ");
		valor = sc.nextDouble();
		System.out.print("Horas trabalhadas: ");
		horas = sc.nextInt();
		
		total = valor * horas;
		
		System.out.println("O pagamento para " + nome + " deve ser " + String.format("%.2f", total));
		
		sc.close();
	}

}

File: Course_01/C Language/EstruturaSequencial/pagamento.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

void clean_entry() {
    char c;
    while ((c = getchar()) != '\n' && c != EOF) {}
}
void read_text(char *buffer, int length) {
    fgets(buffer, length, stdin);
    strtok(buffer, "\n");
}


int main()
{
    char nome[50];
    double valor, horas, salario;

    printf("Nome: ");
    read_text(nome, 50);
    printf("Valor por hora: ");
    scanf("%lf", &valor);
    printf("Horas trabalhadas: ");
    scanf("%lf", &horas);

    salario = valor * horas;

    printf("O pagamento para %s deve ser %.2lf", nome, salario);

    return 0;
}
